### PR TITLE
Relocate chalk package to dependencies

### LINKED ISSUE #138: Chalk should be dependency not just dev
Following the get started calling npx asinit . fails due to missing chalk. npm installing chalk enables it to work as expected.

### PR DESCRIPTION
**Pull request by MaxGraey**
Fix #138

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,9 @@
     "webpack": "^4.8.3",
     "webpack-cli": "^2.1.3"
   },
+  "peerDependencies": {
```
Comment (dcodeIO) [inline package.json]: What's the reason for adding it to peerDependencies instead of dependencies?

Comment (MaxGraey) [reply package.json]: Because chalk need only for devDependencies in host. So user can decide by self when it should located

Comment (dcodeIO) [reply package.json]: Sorry, can't follow. `asinit`, which is essentially a production entity, makes use of it, so shouldn't it be installed normally so a user can start using `asinit` right away without any further steps?

Comment (MaxGraey) [reply package.json]: Ok, I'll move this to normal deps

**Comment by MaxGraey** [comment]
Closed to favour removing chalk instantly.

**Pull request closed without merging**